CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## Changes in 0.5.2 (in development)
+
+- Update of `pyproject.toml'file; package name changed and entry point removed, since xcube
+  plugins auto-recognition is updated. (#39 and xcube-dev/xcube#963)

```

Comment:
Merging of this PR requires a not-yet-released xcube release. 
So let's not merge this before we know the xcube version that fixes it and name that version here.